packages/react-composites/src/composites/CallComposite/pages/UnsupportedBrowser.tsx

```diff
@@ -22,22 +25,32 @@ export type UnsupportedBrowserPageProps = {
  */
 export const UnsupportedBrowserPage = (props: UnsupportedBrowserPageProps): JSX.Element => {
   /* @conditional-compile-remove(unsupported-browser) */
-  const { onTroubleshootingClick } = props;
+  const { onTroubleshootingClick, environmentInfo } = props;
```

Comment:
All of the CC is needed in this file because the `EnvironmentInfo` type is from the beta calling SDK only so will cause errors in the stable flavor if present. This trickles down to all the other CC elements as well.